conf/services/open-banking.cy.yml

```diff
@@ -69,4 +69,5 @@ milestones:
 serviceLastTestedDate: 2021-03-12
 statementVisibility: public
 statementCreatedDate: 2021-03-24
-statementLastUpdatedDate: 2021-03-21
\ No newline at end of file
+statementLastUpdatedDate: 2021-03-24
```

Comment:
This `statementLastUpdatedDate` update looks like it's on the wrong file, shouldn't it have been changed in direct-debit.cy.yml?

Reply:
The date i changed should be on open-banking.cy.yml and